server/views/partials/filters.njk

```diff
@@ -0,0 +1,45 @@
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/select/macro.njk" import govukSelect %}
+
+{% macro filterWrapper(title) %}
+    <div class="search-and-filter">
+        <form action="{{ selfPath }}"
```

Comment:
Is `selfPath` a property meant to be set for all controllers using this? Should it be `currentUrl` instead, which is set for all templates? 

https://github.com/ministryofjustice/hmpps-approved-premises-ui/blob/f2f64b1143a6a0082e1446bda6f66f9ec40cfcae/server/utils/nunjucksSetup.ts#L103

As it stands this results in an empty `action` attribute, which works the same way (submits the form to the current path), but it isn't clear why this variable should be required if that's what we're after!

Reply:
It's a good point that maybe we should just submit to the same URL. In which case, simply omitting the action attribute would be the way to go.